scripts/test_check_backend_patch.py

```
@@ -89,6 +100,26 @@ def test_patch_files_sorted_order() -> None:
         assert names == ["0001-first.patch", "0002-second.patch"], names
 
 
+def test_apply_check_accepts_relative_paths() -> None:
+    with tempfile.TemporaryDirectory() as d:
+        root = Path(d)
+        submodule = root / "submodule"
+        submodule.mkdir()
+        subprocess.run(["git", "init", "--quiet"], cwd=submodule, check=True)
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)